ci-operator/step-registry/cucushift/agent/vsphere/conf/vips/cucushift-agent-vsphere-conf-vips-commands.sh

```diff
@@ -10,17 +10,24 @@ if [[ -z "${LEASED_RESOURCE}" ]]; then
   exit 1
 fi
 
-third_octet=$(grep -oP '[ci|qe\-discon]-segment-\K[[:digit:]]+' <(echo "${LEASED_RESOURCE}"))
-
-if [ "${MASTERS}" -eq 1 ]; then
-  echo "192.168.${third_octet}.4" >> "${SHARED_DIR}"/vips.txt
-  echo "192.168.${third_octet}.4" >> "${SHARED_DIR}"/vips.txt
+if [[ ${LEASED_RESOURCE} == *"vlan"* ]]; then
+  vlanid=$(grep -oP '[ci|qe\-discon]-vlan-\K[[:digit:]]+' <(echo "${LEASED_RESOURCE}"))
+  jq -r --argjson N 2 --arg VLANID "$vlanid" '.[$VLANID].ipAddresses[$N]' /var/run/vault/vsphere-config/subnets.json >> "${SHARED_DIR}"/vips.txt
+  jq -r --argjson N 3 --arg VLANID "$vlanid" '.[$VLANID].ipAddresses[$N]' /var/run/vault/vsphere-config/subnets.json >> "${SHARED_DIR}"/vips.txt
+  jq -r --arg VLANID "$vlanid" '.[$VLANID].machineNetworkCidr' /var/run/vault/vsphere-config/subnets.json >>  "${SHARED_DIR}"/machinecidr.txt
```

Comment:
should we check for missing VLANs in the Vault config? 
replace /var/run/vault/vsphere-config/subnets.json with a  var?